src/samples/EducationStart/ClaimList.tsx

```diff
@@ -83,6 +83,10 @@ export default function ClaimsList(props) {
 
     const target = `${PCore.getConstants().APP.APP}/${container}`;
 
+    sessionStorage.setItem('isComingFromPortal', 'true');
```

Comment:
With introducing session storage here, we should consider if session storage has been disabled or might not be available
`try {
   sessionStorage.setItem('isComingFromPortal', 'true');
   sessionStorage.setItem('isEditMode', 'true');
    sessionStorage.removeItem('stepIDCYA');
    }
   catrch (error) {
   console.log.error("") etc or alternativley switching to local storage or maybe in-memory storage as an object maybe.
   }
`